### PR TITLE
WIP: [Issue #38] Use SI Units

### PR DESCRIPTION
**Pull request by philgutowski**
Will these changes be overwritten when https://github.com/SignalK/signalk-parser-nmea0183/pull/35 gets merged?

### CODE REVIEW

File: test/parsing.js [hunk 1]
```diff
@@ -97,7 +97,7 @@ describe('MWV parser; sentence with true wind data', function() {
                             }
                           },
                           "angleTrueWater": {
-                            "value": -42,
+                            "value": -0.7330382858376184,
```
Comment (philgutowski) [inline test/parsing.js]: @tkurki was I right to update these manually?

File: codecs/MWV.js [hunk 2]
```diff
@@ -45,12 +45,16 @@ Field Number:
 
 var Codec = require('../lib/NMEA0183');
 
+Math.radians = function(degrees) {
+  return degrees * Math.PI / 180;
+};
```
Comment (philgutowski) [inline codecs/MWV.js]: Pull this out for use in the other codecs?

Comment (tkurki) [inline codecs/MWV.js]: As there is no real need for monkey patching let's not do that.

Now that I look at this there's https://github.com/SignalK/signalk-parser-nmea0183/blob/master/lib/NMEA0183.js#L66

I'm not terribly fond of the big (and growing) if block, but to stay in line with the rest of the module I'd say conversions should go here.

File: test/parsing.js (hunk 1 above)
Comment (tkurki) [inline test/parsing.js]: Yes. Check also https://github.com/SignalK/signalk-parser-nmea0183/issues/36 - getting the conversions updated to the proper SI units is priority #1, but fixing precision at the same time would not hurt.

File: codecs/MWV.js (hunk 2 above)
Comment (fabdrol) [inline codecs/MWV.js]: That's where they should go, allowing them to be used across all codecs. @philgutowski do you want to update the rest or shall I go ahead and push a fix for all codecs moving to SI units everywhere?

Comment (philgutowski) [inline codecs/MWV.js]: @fabdrol thanks.  If you have the time, go ahead with it.  This was a good exercise, and I'm starting to better know my way around the system.

**Pull request closed without merging**